templates/logrotate.j2

```diff
@@ -0,0 +1,18 @@
+{{logs_directory}}/*.log
```

Comment:
copied from Charmed MongoDB VM with modifications for pebble service based of pgbouncer